### DISCUSSION TITLE
Is there a way to prevent population of Date and Year folders in Video?

### DISCUSSION THREAD

**Discussion by regul8or** [gerbera/gerbera#2647] [Q&A] [answered]
I understand they loaded from file's metadata which are completely irrelevant for me. Preventing population of them should remove visual clutter, reduce database size and improve autoscan speed

**Comment by KarlStraussberger** [comment, marked as answer]
Hello, thanks for your question, which makes us thinking about other solutions.

There is no real speed or database size improvement to be expected because we need to scan the file to see what metadata are available. File scanning is done with taglib, so no way to pick particular data only.

If you do not need metadata at all, you may turn off virtual layout. I'm right now considering a flag to do this by media type or autoscan directory.

**Reply by regul8or** [reply]
```
select upnp_class, count(*) from mt_cds_object group by upnp_class

NULL 59905
object.container 4803
object.item 2
object.item.videoItem 15189
```

I would say there's at least twice as much records in Date and Year folders as videos themselves. I think removing them reduce database size considerably. Also, not adding unnecessary records to database during scan should improve scan speed.

> you may turn off virtual layout

How do I do this for videos until there's more appropriate solution?

**Reply by KarlStraussberger** [reply]
If you only have videos you can set `virtual-layout="disabled"` (https://docs.gerbera.io/en/stable/config-import.html#scripting). Likewise you can manipulate `addVideo` with js-layout. Or give our new feature of external nfo files (set `metafile` resource and create nfo files) a look and populate metadata properly.

Time is only relevant for first import. Space may only be of interest if you have less than 10MB for the database which I don't think because you can configure it to use the same disk as your videos.

**Reply by regul8or** [reply]
Thanks, with disabled `virtual-layout` it looks very like I expect it to be presented on my TV
